media/capture/content/android/java/src/org/chromium/media/ScreenOffloading.java

```diff
@@ -193,11 +194,16 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             mResultCode = resultCode;
             mResultData = data;
             changeCaptureStateAndNotify(CaptureState.ALLOWED);
+            if (OffloadingUtils.IsServiceOffloading()) {
+                ApplicationStatus.getLastTrackedFocusedActivity().moveTaskToBack(true);
```

Comment:
Just my opinion. How about notify that screen capture is started using Toast?

Reply:
We have screen recording msg on top of android once recording starts. Therefore, I think we don't need toast.